tests/istio_testcases/expected_output/connectivity_bookinfo_ratings_allow_any_sidecar.txt

```diff
@@ -0,0 +1,21 @@
+For connections of type TCP, final fw rules for query: connectivity-allow-any, config: allow-all-sidecar:
+src: 0.0.0.0/0 dst_ns: [default] dst_pods: [*] conn: All connections
+src_ns: [default] src_pods: [*] dst: *.newrelic.com conn: TCP 443
+src_ns: [default] src_pods: [*] dst: *.slack.com conn: TCP 443
+src_ns: [default] src_pods: [*] dst: *.wikipedia.org conn: TCP 443
+src_ns: [default] src_pods: [*] dst: 0.0.0.0/0 conn: All connections
+src_ns: [default] src_pods: [*] dst: api.dropboxapi.com conn: TCP 443
+src_ns: [default] src_pods: [*] dst: api.facebook.com conn: TCP 443
+src_ns: [default] src_pods: [*] dst: www.googleapis.com conn: TCP 443
+src_ns: [default] src_pods: [*] dst_ns: [default] dst_pods: [*] conn: All connections
+src_ns: [default] src_pods: [app!=ratings] dst: *.newrelic.com conn: All connections
+src_ns: [default] src_pods: [app!=ratings] dst: *.slack.com conn: All connections
+src_ns: [default] src_pods: [app!=ratings] dst: *.wikipedia.org conn: All connections
+src_ns: [default] src_pods: [app!=ratings] dst: api.dropboxapi.com conn: All connections
+src_ns: [default] src_pods: [app!=ratings] dst: api.facebook.com conn: All connections
+src_ns: [default] src_pods: [app!=ratings] dst: www.googleapis.com conn: All connections
```

Comment:
actually this may be a bug.
the sidecar in this test selects only the app=ratings.
In computing allowed connections for a sidecar, I consider the ports of the DNSEnry peers (this is why we see above that from ratings to same dns names the conns allowed is TCP 443 and not All connections)
Since there is no policy selecting the other workloads they assume that `all connections` are allowed to these DNSEntry peers and not only the ports declared in the ServiceEntry.

I will confirm on live cluster that the connections anyhow should be to the declared ports only
then I need to find a "global" way to tell that the "allowed conns" to a DNSEntry peer is only on the declared ports (not only in sidecar's allowed_connections def)

Reply:
fixed